### PR TITLE
particles['x'] and ['y'] now always in grid units

### PR DESCRIPTION
**Pull request by tobson**
I think it's best (least confusing) for the coordinates in the particle
array not to measured in different units in different parts of the code.

### CODE REVIEW

**Comment by tobson** [comment]
@tberlok Let's do a rebase and merge if you agree with this change.

File: tests/test_fastwave.py
Comment (tberlok) [inline tests/test_fastwave.py:107]: I don't think this will work? What is x and y here?

Comment (tberlok) [reply tests/test_fastwave.py:107]: Nevermind the other comment. This should work, I guess.

**Review by tberlok — CHANGES_REQUESTED** [review]
You should add the same change in particles.py line 119-120 then.

**Comment by tberlok** [comment]
I am going to bed, feel free to rebase and merge after adding the changes to the DensityPertubation class.

File: tests/test_fastwave.py
Comment (tobson) [reply tests/test_fastwave.py:107]: I changed lines 119-120. Thanks for catching that.

**Comment by tobson** [comment]
It looks like you'll now need to update your review and approve of the changes before this can be merged. I can also dismiss your review, but that seems like poor form...

**Review by tberlok — APPROVED** [review]

**Pull request merged**
